src/Latte/Parser/VarTypeParser.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Rector\Nette\Latte\Parser;
+
+use Nette\Utils\Strings;
+use Rector\Nette\ValueObject\LatteVariableType;
+
+final class VarTypeParser
+{
+    /**
+     * @return LatteVariableType[]
+     */
+    public function parse(string $content): array
+    {
+        $pattern = '#{varType (?P<type>.*?) \$(?P<variable>.*?)}#';
```

Comment:
Regexes are hell to debug, when something breaks in the future.

Could you extract a constant with https://regex101.com/ link like we use in Rector core?

This should be reported by PHPStan, but it was missed :/

Reply:
Sure, I'll add